.goreleaser.yml

```diff
@@ -77,3 +77,17 @@ changelog:
     exclude:
     - '^docs:'
     - '^test:'
+    
+snapcrafts:
+  -
+    replacements:
+      386: i386
+      amd64: x86_64
+    summary: The MongoDB Command Line Interface
+    description: The MongoDB Command Line Interface (mongocli) is a tool for managing your MongoDB cloud services, like MongoDB Atlas, MongoDB Cloud Manager, and MongoDB Ops Manager.
+    grade: stable
```

Comment:
can you please change this to `devel` as we are not GA yet?

Reply:
done!